docs/how-to-run-ohmyglb.md

```diff
@@ -0,0 +1,105 @@
+# How to run Oh My GLB locally
+
+- [Overview](#overview)
+- [Environment prerequisites](#environment-prerequisites)
+- [Running project locally](#running-project-locally)
+- [See if everything is well installed](#see-if-everything-is-well-installed)
+- [See if everything running smoothly](#see-if-everything-running-smoothly)
+- [Cleaning](#cleaning)
+
+## Overview 
+The document helps contributors participating on GSLB to install local environment. 
+It is not intend to force you to strictly follow the document, but it can help you to 
+avoid troubles related setup local environment. 
+  
+
+![](./images/specialist2.jpg)
+
+## Environment prerequisites
+
+ - install **GO 1.14**
+    - ensure you have standard GO folder structure created and $GOPATH, $GOROOT are properly set. 
+ 
+ - install **GIT**     
+    - ensure SSH key is set for your account; see https://github.com/settings/keys  
+    
+ - install **gnu-sed** if you don't have it  
+    - update $PATH in `~/.bash_profile` by sed path `export PATH=/usr/local/opt/gnu-sed/libexec/gnubin:$PATH` 
+   
+ - install **Docker**
+    - ensure you are able to push/pull from your docker registry
+    - to run multiple clusters reserve 8GB of memory  
+    
+      ![](./images/docker_settings.png)
+
+ - install **Kubectl** to operate clusters
+     
+ - install **Helm** to get charts
```

Comment:
Explicitly note the version of Helm and perhaps a link to install, given the Helm 2 and 3 options currently prevalent.